### PR TITLE
Put "Enable only for specific groups" into a new div

### PR DESCRIPTION
**Pull request by skjnldsv**
I think this is better looking and much cleared now.
Before
capture d ecran_2016-12-02_23-53-02
After
capture d ecran_2016-12-02_23-52-45

### CODE REVIEW

**Comment by MorrisJobke** [comment]
Tested and works 👍 

cc @nextcloud/designers please review

File: settings/templates/apps.php [hunk 1]
```diff
@@ -125,10 +125,10 @@
 	<input class="update hidden" type="submit" value="<?php p($l->t('Update to %s', array('{{update}}'))); ?>" data-appid="{{id}}" />
 	{{#if active}}
 	<input class="enable" type="submit" data-appid="{{id}}" data-active="true" value="<?php p($l->t("Disable"));?>"/>
-	<span class="groups-enable">
+	<div class="groups-enable">
 		<input type="checkbox" class="groups-enable__checkbox checkbox" id="groups_enable-{{id}}"/>
 		<label for="groups_enable-{{id}}"><?php p($l->t('Enable only for specific groups')); ?></label>
-	</span>
+	</div>
 	<br />
```
Comment (eppfel) [inline settings/templates/apps.php]: Could we get rid of this `<br />` then?

**Comment by eppfel** [comment]
It's an improvement. But these tiles/cards have to be redesign completely.

I rebased on #2434 and the margin/padding is not right. At least some space to the `enable` button.
bildschirmfoto 2016-12-03 um 00 33 04

**Comment by skjnldsv** [comment]
@eppfel You're right!
For the entire redesign, if I have an idea I will try to propose it before nc11 otherwise, we'll see for nc12!

**Comment by skjnldsv** [comment]
@eppfel rebasing is fine here! :/
capture d ecran_2016-12-03_08-38-02

**Comment by MariusBluem** [comment]
Much better now, and definetly something we want to see in Nextcloud 11 👍 
...this is not messing up the entire page - and works without any Problems for me.

**Comment by eppfel** [comment]
Yeah, the redesign will be in NC12. Think it should be more like the appstore looks now...

I'll add some CSS for the spacing. Might take me till tomorrow.

**Comment by skjnldsv** [comment]
I was thinking to something like that
capture d ecran_2016-12-04_09-50-59

**Comment by rullzer** [comment]
Lets do this!
Please leave the final overhaul to 12 :wink: 

:+1:

**Comment by jancborchardt** [comment]
@skjnldsv proposal looks like a good improvement! We only need to make sure the label »Official« doesn’t look like a button and with the checkmark like an »Enabled« state. Can you open a new issue about the improvement with Nextcloud 12 as milestone? :)

**Pull request merged**